doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -560,6 +560,10 @@ Cellular samples
     * Handling of L4 disconnect where CoAP connection is paused and socket is kept open, then resumed when L4 reconnects.
     * Checking in CoAP FOTA and shadow polling threads to improve recovery from communications failures.
 
+  * Changed:
```

Comment:
since this is just one item, i don't think it needs a heading.
i believe `Updated:` is the standard heading when there are multiple items.
i don't know if there are any actual rules about this or just convention.

Reply:
Fully agree with Justin.